airbyte-integrations/connectors/source-typeform/source_typeform/spec.json

```diff
@@ -17,6 +17,15 @@
         "type": "string",
         "description": "The API Token for a Typeform account.",
         "airbyte_secret": true
+      },
+      "form_ids": {
+        "title": "Form ids",
+        "description": "List of form ids to sync from. By default all forms will be synced, you can use this parameter to select set of forms",
```

Comment:
just a rephrasing, also where can a user find their form IDs? can we add instructions in the docs for them and add an `<a>` here? 
```suggestion
        "description": "When this parameter is set, the connector will replicate data only from the input forms. Otherwise, all forms in your Typeform account will be replicated. See <HERE> for instructions on how to find your form IDs",
```

Reply:
@sherifnada unfortunately I didn't find separate instruction where can a user find their form IDs.

Reply:
@vitaliizazmic it is right in the do
<img width="759" alt="Screenshot 2021-10-05 at 18 37 58" src="https://user-images.githubusercontent.com/1619536/136055853-d20551de-16e9-4a01-abbc-997677e63124.png">